docs/src/_includes/components/navigation.html

```diff
@@ -22,4 +22,19 @@
             {% include 'components/nav-search.html' %}
         </div>
     </div>
+    <noscript>
+        <div class="docs-nav-panel" >
+            <ul id="nav-list">
+                {% for item in navigation.items %}
+                <li>
+                    <a href="{{ item.url }}" {% if item.text == 'Docs' %} aria-current="true"{% endif %}>{{ item.text }}</a>
+                </li>
+                {% endfor %}
+            </ul>
+        </div>
+        <div class="mobile-only">
+            {% include 'components/nav-version-switcher-noscript.html' %}
+            {% include 'components/nav-search.html' %}
+        </div>
+    </noscript>
```

Comment:
Can we avoid duplicating the docs nav panel, like we avoided duplicating the docs index list? (https://github.com/eslint/eslint/pull/15926#discussion_r883910216)

Reply:
yes, done.